### PR TITLE
Add Gatsby

### PR DESCRIPTION
**Pull request by BenRauzi**
Apologies for the huge PR.

Key things from the previous iteration you reviewed:
- Added Santisation
- Added Netlify CMS with previews
- Moved to JS configuration of the CMS for reusability
- Modified file structure to work better as the site is modified
- Added alias imports for tidier imports
- Separated css to make it easier to work with

Further improvements to make over coming days:
- Expand CMS to cover whole site and provide rich-text editing
- Potentially move to CSS modules to prevent class-name duplication issues
- API mocks
- Documentation

### CODE REVIEW

**Comment by BenRauzi** [comment]
https://openlawnz-gatsby-new.netlify.app

File: package.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-react": "^7.21.5",
+    "gatsby-plugin-eslint": "^2.0.8",
+    "prettier": "2.1.2"
+  },
+  "homepage": "https://openlaw.nz/",
+  "keywords": [
+    "gatsby"
+  ],
+  "license": "0BSD",
```
Comment (williamparry) [inline package.json:52]: This will need to be GPL

File: gatsby-config.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `${__dirname}/content`,
+        name: `assets`,
+      },
+    },
+    {
+      resolve: `gatsby-plugin-google-analytics`,
+      options: {
+        //trackingId: `ADD YOUR TRACKING ID HERE`,
```
Comment (williamparry) [inline gatsby-config.js:33]: Do we have this?

Comment (BenRauzi) [reply gatsby-config.js]: Added c7842bc

**Comment by williamparry** [comment]
Great work, @BenRauzi!

**Pull request merged**